src/libcrun/linux.c

```diff
@@ -3476,10 +3476,10 @@ libcrun_run_linux_container (libcrun_container_t *container, container_entrypoin
   if (UNLIKELY (ret < 0))
     return ret;
 
-  get_private_data (container)->unshare_flags = init_status.all_namespaces;
+  get_private_data (container)->unshare_flags = init_status.namespaces_to_unshare;
 #if CLONE_NEWCGROUP
   /* cgroup will be unshared later.  Once the process is in the correct cgroup.  */
-  init_status.all_namespaces &= ~CLONE_NEWCGROUP;
+  init_status.namespaces_to_unshare &= ~CLONE_NEWCGROUP;
```

Comment:
Does `namespaces_to_unshare` contains `namespaces` specified without `path` ? I am still a bit confused about this.

Reply:
Tests a failing for: when `uts` namespace is specified by a `path`.